printrun/utils.py

```diff
@@ -118,14 +150,34 @@ def lookup_file(filename, prefixes):
     return filename
 
 def pixmapfile(filename):
+    '''
+    Get the full path to filename by checking in standard icon
+    ("pixmaps") directories (See the lookup_file function's
+    documentation for behavior).
+    '''
     shared_pixmaps_dir = os.path.join(DATADIR, 'pixmaps')
-    return lookup_file(filename, [shared_pixmaps_dir])
+    local_pixmaps_dir = os.path.join(
+        os.path.dirname(os.path.dirname(sys.argv[0])),
+        "share",
+        "pixmaps"
+    )  # Used by pip install
+    pixmaps_dirs = [shared_pixmaps_dir, local_pixmaps_dir]
+    return lookup_file(filename, pixmaps_dirs)
 
 def sharedfile(filename):
+    '''
+    Get the full path to filename by checking in the shared
+    directory (See the lookup_file function's documentation for behavior).
+    '''
     shared_pronterface_dir = os.path.join(DATADIR, 'pronterface')
     return lookup_file(filename, [shared_pronterface_dir])
 
 def configfile(filename):
+    '''
+    Get the full path to filename by checking in the
+    standard configuration directory (See the lookup_file
+    function's documentation for behavior).
+    '''
     return lookup_file(filename, [os.path.expanduser("~/.printrun/"), ])
```

Comment:
Silly question maybe and not directly related to this PR anyway. Why is this function checking for a configuration file in `~/.printrun`? AFAIK pronterface/pronsole look for a config file in either `~/.config/Printrun/pronsolerc`, `~/.pronsolerc` or `~/printrunconf.ini`.

Reply:
I suggest making an issue for that. The usage of the term `configfile` is ambiguously used as a global function and local variable, only ever used in pronterface.py, and it is only called as a function once, in a way that it isn't clear what is supposed to happen:
`exec(compile_file(configfile("custombtn.txt")), customdict)`

Reply:
Here you go (As soon as someone figures out what custombtn.txt is and whether arbitrary code execution is even safe or expected here, we can figure out what to do): https://github.com/kliment/Printrun/issues/1286

Reply:
Thanks again, I like the new issue approach. I'll close this revision then.